_posts/en/resources/2019-02-12-tinkercad-resistor-led.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Make sure you are always forking from the latest version of the origin/master branch

Reply:
Hi @marc-ed-raffalli  The issue here was that I'd started authoring the branch whilst the master was updated whilst the branch was being authored. I needed to merge changes from master to the branch prior to checkin of the branch ? If you want to follow a different workflow i certainly can. Let me know. Thanks.